### PR TITLE
provide readme for containers module

### LINKED ISSUE #10092: Provide readme to container submodule of quarkus dist
### Description

... covering at least how to build an image locally from the dist.

### Discussion

_No response_

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by DGuhr**
seems to accidentally gone missing when moving from containers repo

Closes #10092

### CODE REVIEW

File: quarkus/container/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+RUN keytool -genkeypair -storepass password -storetype PKCS12 -keyalg RSA -keysize 2048 -dname "CN=server" -alias server -ext "SAN:c=DNS:localhost,IP:127.0.0.1" -keystore conf/server.keystore
+
+RUN ./bin/kc.sh build --db=postgres && ./bin/kc.sh start-dev --db-url=jdbc:postgresql://$DB_HOST/keycloak --db-username=keycloak --db-password=password
+```
+NOTE: Note that we are using separated `RUN` steps for illustrative purposes, ideally you should reduce the number of layers and execute all steps in a single one.
+
+In the example above, Keycloak is set to use e a PostgreSQL database in development mode. You can for sure also use environment variables for the configuration and build an optimized image for better startup time. For more information, see the [Running Keycloak in a container guide](https://www.keycloak.org/guides/server/containers).
+
+
+## Build the image
```
Comment (stianst) [inline quarkus/container/README.md:62]:
```suggestion

```

I'd remove everything around running and extending the image, as that should be covered by the guides. This should just be about building it.

File: quarkus/container/README.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+It is possible to download the Keycloak distribution from a URL:
+
+    docker build --build-arg KEYCLOAK_DIST=http://<HOST>:<PORT>/keycloak-<VERSION>.tar.gz . -t <YOUR_TAG>
+
+Alternatively, you need to build the local distribution first, then copy the distributions tar package in the `containers` folder and point the build command to use the image:
+
+    cp $KEYCLOAK_SOURCE/quarkus/dist/target/keycloak-<VERSION>.tar.gz .
+    docker build --build-arg KEYCLOAK_DIST=keycloak-<VERSION>.tar.gz . -t <YOUR_TAG>
```
Comment (stianst) [inline quarkus/container/README.md:71]: Was wondering if we should add a pom.xml or a script that does this part?

Comment (DGuhr) [reply quarkus/container/README.md:13]: not in this issue, i guess? in general a good idea, but might be evaluated first if we want to do it that way or another? don't know.

File: quarkus/container/README.md (hunk 1 above)
Comment (DGuhr) [reply quarkus/container/README.md]: done

**Review by pedroigor — APPROVED** [review]

**Review by pedroigor — APPROVED** [review]

**Review by abstractj — APPROVED** [review]
LGTM

**Review by abstractj — APPROVED** [review]
LGTM

**Review by stianst — APPROVED** [review]

**Comment by sschu** [comment]
@DGuhr I know I am a little late to the party, but the link to the Keycloak containers guide is broken.

**Comment by DGuhr** [comment]
@sschu thanks for noticing! Yers, right link should be https://www.keycloak.org/server/containers - will provide a PR.

**Pull request merged**